e2echart/e2e-chart-template.html

```diff
@@ -313,6 +357,68 @@ <h5 class="modal-title">Resource</h5>
         return [item.locator, ` (${roles})`, m];
     }
 
+    function etcdLogs(item) {
+        let nodeVal = 'noNode'
+        if (item.tempStructuredLocator &&
+            item.tempStructuredLocator.keys &&
+            item.tempStructuredLocator.keys.hasOwnProperty('etcd-member') &&
+            item.tempStructuredLocator.keys.hasOwnProperty('node')) {
+            if (item.tempStructuredLocator.keys['node'].length > 0) {
+                nodeVal = item.tempStructuredLocator.keys['node']
+            }
+        }
+        // See if this is a leader change interval.
+        const [isLeaderFoundOrLost, foundOrLost] = isEtcdLeaderFoundOrLost(item)
+        if (isLeaderFoundOrLost) {
+
+            // For leader-found, a 'term' is expected to be present; this line attempts to
+            // get the value of 'term' and if not there, we'll default to ''.
+            let term = item.tempStructuredMessage?.annotations?.['term'] ?? '';
+            if (term.length == 0) {
+                // For leader-lost, leader-elected, and leader-missing, term is in the humanMessage.
+                // Ensure it exists and if not, just default to 'noTerm'
+                term = item.tempStructuredMessage?.humanMessage ?? 'noTerm';
+            }
+            nodeVal = item.tempStructuredLocator.keys['node']
+
+            let leaderColor ="EtcdUnknown"
+            switch (foundOrLost) {
+                case "leader-found":
+                    leaderColor = "EtcdLeaderFound";
+                    break;
+                case "leader-lost":
+                    leaderColor = "EtcdLeaderLost";
+                    break;
+                case "leader-elected":
+                    leaderColor = "EtcdLeaderElected";
+                    break;
+                case "leader-missing":
+                    leaderColor = "EtcdLeaderMissing";
+                    break;
```

Comment:
I would prefer to see this logic moved to origin itself when generating the interval, as these Reason values don't match the syntax of any others, they're always CamelCase. 

Color will be tricky when we get to porting the whole file.